Sources/SwerverCore/Response/ResponseBuilder.swift

```diff
@@ -0,0 +1,16 @@
+import Foundation
+
+public class ResponseBuilder {
```

Comment:
Be careful calling this a "builder". A "FooBuilder" in object oriented programming is often used to designate some class that follows the builder pattern.

Reply:
This is just saying that the responsebuilder class will construct a response object. perhaps ResponseConstructor is better?

Reply:
fixed by using builder pattern